lib/eventboss/cli.rb

```diff
@@ -0,0 +1,150 @@
+require 'rubygems'
+require 'dotenv'
+require 'optparse'
+require 'yaml'
+require 'erb'
+require 'singleton'
+require_relative '../eventboss'
+
+module Eventboss
+  class CLI
+    include Singleton
+
+    attr_accessor :options
+
+    DEFAULT_OPTIONS = {
+      require: '.'
+    }
+
+    def initialize
+      self.options = DEFAULT_OPTIONS.dup
+    end
+
+    def parse(args = ARGV)
+      parse_options(args)
+      load_config_file
+    end
+
+    def run
+      boot_system
+
+      Eventboss.logger.info('Starting eventboss...')
+      Eventboss.logger.info('Active Listeners:')
+      Eventboss.logger.info(Eventboss::QueueListener.list_active.to_s)
+
+      Eventboss.launch
+    end
+
+    private
+
+    def boot_system
+      Dotenv.load
```

Comment:
shouldn't dotenv be used only on development/test? i'm not sure if anyone used in on prod?

Reply:
It was like that before, I just moved it to boot_system method. I wouldn't like to extend this PR even more. Wdyt @managr ?

Reply:
It makes sense to remove in the future IMO, unfortunately we'll have to inherit it for now

Reply:
Well, why? Configure, in my opinion, is just another way of configuring it. Right now we'll have 3 options:
* env variables (which is really flexible, especially if sth differ per environment)
* configure (which could set things programmatically) 
* conf file which is static configuration